### PR TITLE
data thon post

### PR DESCRIPTION
**Pull request by mbarger14**
data-thon post without pictures for review. not ready to merge 
Screenshot 2023-12-19 at 9 40 11 AM

Screenshot 2023-12-19 at 9 40 46 AM

Screenshot 2023-12-19 at 9 41 26 AM
Screenshot 2023-12-19 at 9 41 47 AM
Screenshot 2023-12-19 at 9 42 08 AM
Screenshot 2023-12-19 at 9 42 29 AM

### CODE REVIEW

**Review by tfreebern2 — COMMENTED** [review]
@mbarger14 Do we want to push an empty blog?

**Comment by mbarger14** [comment]
@tfreebern2 its fixed. please dont push it yet

**Comment by tfreebern2** [comment]
> @tfreebern2 its fixed. please dont push it yet

You can mark the PR as a Draft to prevent it from being approved/merged.

**Comment by mbarger14** [comment]
> > @tfreebern2 its fixed. please dont push it yet
> 
> You can mark the PR as a Draft to prevent it from being approved/merged.

Awesome, thank you!

File: _posts/2023-12-21-third-annual-data-thon [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## IV. Showcasing Veteran-Centric Solutions
+
+The resulting outcome is not merely a solution, but a versatile toolset poised to bring tangible benefits to the VA.
+
+1. **API Integration for Decision Tree Insights:** The Decision Tree-based model is not a static achievement confined to competition metrics. Instead, it emerges as a dynamic resource that can be seamlessly integrated into the VA's operations. The team envisioned the model serving as an API, allowing for the exportation of decision trees to Python tables. This functionality ensures that decision-makers within the VA can access and leverage the insights derived from the model in a manner that aligns with their existing workflows.
+
+2. **ETL for Visualization Tools:** The clean and transformed data served a dual purpose. Beyond the modeling phase, this data became a valuable Extract, Transform, Load (ETL) resource. It fed into visualization tools such as Power BI and Tabula, providing a visual narrative of predictive analytics. This not only enhanced the interpretability of results but also empowered VA stakeholders to make informed decisions based on rich, accessible insights.
```
Comment (dhall) [inline _posts/2023-12-21-third-annual-data-thon:49]: Is `Tabula` supposed to be `Tableau` in this paragraph?

Comment (bastosmichael) [reply _posts/2023-12-21-third-annual-data-thon]: It was Tableau thanks for catching...

**Review by bastosmichael — APPROVED** [review]

**Pull request merged**